### ISSUE TITLE
Server Becoming Unresponsive

### ISSUE THREAD

**Issue by Dizzy2122** [xJon/The-1.12.2-Pack#652]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
yes
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
yes
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
yes

**Information required**
1. https://www.dropbox.com/s/nlo59dr7wuosxwh/latest.log?dl=0  -- Most recent latest.log, uploaded to dropbox
2. Is Optifine installed or any other additional mods: no

**Issue description**
I've been hosting a server for myself and a few friends for quite some time now, actually this is my 3rd server over the past few years for this but this is the first time I've been dealing with this issue. Server ran flawlessly for the first month or two but in recent days its been locking up on me more, and now its almost every 10-30 minutes. It generates no crash log, it doesn't shut down, it just goes completely unresponsive. Commands won't run in the terminal when this happens too, I always try to do a stop command when it happens but unfortunately it has to be forcibly terminated and restarted. Any help or suggestions on what's causing this would be great. This is the most work we've put into a server on this pack and I'd hate to loose it all.

**Comment by Dizzy2122** [comment]
I forgot to mention, I have pulled the world folder out of the server, freshly wiped and installed a clean server and imported our world back in and get the same results.

**Comment by xJon** [comment]
You should have backups of the world so you can consider rolling back before this started happening.
Additionally, you could try to use profiling mods, such as [LagGoggles ](https://www.curseforge.com/minecraft/mc-mods/laggoggles)& [Spark](https://www.curseforge.com/minecraft/mc-mods/spark).

**Comment by Dizzy2122** [comment]
@xJon Perfect thank you, I'll throw one of those in and report back with a ton of info or see if I can at least narrow it down, I'll give my backups a check too, although it has been happening since quite early on, just a lot less frequent haha. But thank you I will report back soon!

**Comment by Dizzy2122** [comment]
@xJon https://www.dropbox.com/s/4x1gtzax0xf3zq9/3-18-20%20latest.log?dl=0 here's what I pulled after using spark. Don't know much about what I'm looking at, but I started the server, ran the commands, joined the server, ran the commands, stepped away from my PC while everything was running, had my friend join and that's when it went unresponsive. The log doesn't show the commands I tried while it was unresponsive, including the stop command but it had to be forcibly closed.

Thank you again for the assistance.
